frontend/schema.graphql

```diff
@@ -1,4 +1,4 @@
-﻿schema {
+schema {
```

Comment:
Switching to dotnet 9.0 seems to have caused `task api:generate-gql-schema` to no longer produce a U+FEFF character at the start of the file. This is just fine by me; I'd like the UTF-8 BOM sequence to disappear entirely from the Internet. It's only useful in the case of buggy software, and it causes more problems than it solves.